config.fullstack.test.yaml

```diff
@@ -253,8 +253,9 @@ info:
 services:
   - name: restbase
     module: hyperswitch
-    conf:
-      port: 7231
+    conf: &rb_conf
```

Comment:
In reality, we need two services in this config; one on port 7231 and the other 7233. They ought to have the same config.

Reply:
See comment: 

>  In production we expose both 7231 and 7233 ports from fullstack RB.

I can't do it in here because of fsfe test configuration. 2 processes would use the same port in that case.

Reply:
Ah, missed it! Thnx!